source/administration/config-settings.rst

```diff
@@ -1543,9 +1543,14 @@ Show @channel and @all confirmation dialog
 
 Enable Email Notifications
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 **True**: Enables sending of email notifications.
 
-**False**: Disables email notifications for developers who may want to skip email setup for faster development. To remove the **Preview Mode: Email notifications have not been configured** banner, also set **Enable Preview Mode Banner** to ``false``. If this setting is set to ``false`` and the SMTP server is set up, account related emails will be sent for user account changes such as password, email, username, user token, MFA, and sign in type changes.  Email invitations and account deactivation emails will also be sent.
+**False**: Disables email notifications for posts. This is useful for developers who may want to skip email setup for faster development. In order to remove the **Preview Mode: Email notifications have not been configured** banner, you should also set **Enable Preview Mode Banner** setting to false.
+
+If this setting is set to false and the SMTP server is set up, account related emails (such as password, email, username, user token, MFA, and other authentication related changes) will be sent regardless of this setting. 
```

Comment:
Should we add a note that this is going to be fixed in a later version? Because current versions don't send password reset mails, and customers might be confused.

Reply:
I think this is [resolved](https://github.com/mattermost/mattermost-server/issues/14977#issuecomment-665471031) for this specific setting right?

Reply:
Yeah it is, but only for customers with >=5.28. So they might be confused if they are on a previous version. So a note that this is fixed from 5.28 onwards might be clearer.

Reply:
@agnivade I assume you mean a different version. I just tested on v5.25 and password reset emails are still sent - even with this setting set to false.

Reply:
Sorry disregard. At that point of time I thought that reset emails were part of the bug, but they aren't. Thanks for checking.